accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/authorizableinstaller/impl/AuthInstallerUserManagerPrefetchingImpl.java

```diff
@@ -51,7 +52,7 @@
     private static final Logger LOG = LoggerFactory.getLogger(AuthInstallerUserManagerPrefetchingImpl.class);
 
     private final UserManager delegate;
-    private final Map<String, Authorizable> authorizableCache = new HashMap<>();
+    private final Map<String, Authorizable> authorizableCache = new CaseInsensitiveMap();
```

Comment:
what about the other maps below?

Also a reference towards https://github.com/apache/jackrabbit-oak/blob/cde907088a5c460178fd714cb4edc141cd45e23d/oak-core/src/main/java/org/apache/jackrabbit/oak/security/user/AuthorizableBaseProvider.java#L84 in some comments helps to understand the issue better

Reply:
I don't know if the other maps need an adjustment. They don't seem to have the same issue since they don't fall back to the delegate behavior. Would you recommend to change those to CaseInsensitiveMap as well? Any other adjustments needed?